docs/README.md

```diff
@@ -26,7 +26,7 @@ Core is the flagship application of VATSIM UK. Originally designed to handle Sin
 
 |     Env    |              URL              |        Status        |
 |------------|-------------------------------|----------------------|
-| Production | https://core.vatsim.uk        | ![production_status] |
+| Production | https://vatsim.uk             | ![production_status] |
```

Comment:
I’d prefer if this referenced our full production APP_URL of www.vatsim.uk

Reply:
Thanks, do we have a status badge for the alpha env?

Reply:
https://vatsim-uk.deploybot.com/badge/77558059841720/140917.png